pgo/cmd/create.go

```diff
@@ -264,7 +264,7 @@ func init() {
 	//	createClusterCmd.Flags().BoolVarP(&ArchiveFlag, "archive", "", false, "Enables archive logging for the database cluster.")
 	createClusterCmd.Flags().StringVarP(&PgpoolSecret, "pgpool-secret", "", "", "The name of a pgpool secret to use for the pgpool configuration.")
 	createClusterCmd.Flags().BoolVarP(&MetricsFlag, "metrics", "", false, "Adds the crunchy-collect container to the database pod.")
-	createClusterCmd.Flags().BoolVarP(&AutofailFlag, "autofail", "", false, "If set, will cause autofailover to be enabled on this cluster.")
+	createClusterCmd.Flags().BoolVarP(&AutofailFlag, "disable-autofail", "", false, "If set, will cause autofailover to be enabled on this cluster.")
```

Comment:
I think you mean the opposite in this description string, right?

Reply:
I would double up on this, and just say let's rename it to `DisableAutofailFlag`.

For the CLI, this seems fair enough to ensure we capture the correct functionality internally and not rely on `!Autofail`

For the API we can leave it as `Autofail` given we are saying `true`/`false` there.